### DISCUSSION TITLE
help: Is there any way I can customize this request domain name?

### DISCUSSION THREAD

**Discussion by blockmood** [stoplightio/elements#2353] [Q&A]
![1681365034060](https://user-images.githubusercontent.com/22879524/231665645-7e818255-355d-40b4-9daa-93881b183d39.jpg)

**Comment by CleanMatyx** [comment]
To customize the request domain name, you need to use the AddServer method within your Swagger generation setup. This allows you to explicitly set the Url that the API client should target, overriding the default behavior (which usually defaults to the host serving the Swagger UI). You can add multiple servers to provide options for different environments (e.g., Local, Proxy, Production). 

`SwaggerConfig.AddServer(
    New OpenApiServer {
        Url = "https://your-custom-domain.com/",
        Description = "Custom API Domain"
    }
)`
